### ISSUE TITLE
Search feature on validator list page

### ISSUE THREAD

**Issue by wakqasahmed** [tuum-tech/vouch-capsule#81]
Once we have a good number of validators for each credential type, it would be difficult for the user to find their preferred validator from the list.

The search feature (by validator's name) would be handy.

**Comment by kpachhai** [comment]
I agree. good suggestion

**Comment by mfaizanatiq** [comment]
We can implement the same search feature as we did in the Assist app for DIDS. 

Search and highlight the keywords in the list OF validators. 

Are planning to keep options for pro or paid validators to stay on the top of the list?

**Comment by kpachhai** [comment]
Yes, there are two kinds of validators - ones that are running an actual validator node in the backend(this is more robust and trustworthy) and ones that are validating directly via Vouch app. We'll have to put the validator nodes at the top of the list.

**Comment by kpachhai** [comment]
As far as how the search and highlight works, yea that makes sense to do that on this search feature as well
